build.sbt

```diff
@@ -0,0 +1,36 @@
+val nameStr = """codacy-metrics-scala"""
```

Comment:
Check this file as example: https://github.com/codacy/codacy-metrics-gocyclo/blob/master/build.sbt